src/components/navbar/Navbar.js

```diff
@@ -1,22 +1,81 @@
 import React from 'react';
 import { translate } from 'react-i18next';
 import 'i18n/i18n';
+import { slide as Menu } from 'react-burger-menu';
 import styles from './navbar.scss';
 
-const Navbar = ({ t }) => (
-  <nav className={styles.navigationBar}>
-    <div className={styles.inner}>
-      <a href="#about" className={styles.navLinks}>
-        {t('About')}
-      </a>
-      <a href="#mission" className={styles.navLinks}>
-        {t('Mission')}
-      </a>
-      <a href="#contact" className={styles.navLinks}>
-        {t('Contact')}
-      </a>
+const Navbar = ({ t }) => {
+  const BurgerMenustyles = {
```

Comment:
Can we move these styles to a `scss` file?

Reply:
I tried moving it to _scss_ then importing back but apparently, for some reason, the styles aren't not getting applied. I'll try updating this branch tomorrow.

Reply:
Is not loading because you are not using them in the react component, consider doing something like this:

```
import BurgerMenuStyles from './burgermenu.scss';
```

We are using css modules, therefore the final bundle only includes what we use inside the react components, by importing the styles and adding the classnames in an object then using it where you need, webpack will include them in the final bundle

Please also use lowercase for the scss files.

Reply:
Hey, I tried that as well.
If I do it the following way:
`import burgerStyles from './burgerMenu.scss';`
and 
`console.log(burgerStyles)`
Its throwing the following output: 
![image](https://user-images.githubusercontent.com/11453045/46619014-969bbd80-cb3e-11e8-83d9-7d8c300bc1f3.png)

whereas we except it to be the following way:
`{
    bmBurgerButton: {
      position: 'fixed',
      width: '36px',
      height: '30px',
      left: '36px',
      top: '36px',
    },
    bmBurgerBars: {
      background: '#21785b',
    },
    bmCrossButton: {
      height: '24px',
      width: '24px',
    },
    bmCross: {
      background: '#979797',
    },
    bmMenu: {
      background: '#afe9d6',
      padding: '2.5em 1.5em 0',
      fontSize: '1.15em',
    },
    bmMorphShape: {
      fill: '#373a47',
    },
    bmItemList: {
      color: '#b8b7ad',
      padding: '0.8em',
    },
    bmItem: {
      display: 'block',
      padding: '1.2em',
    },
    bmOverlay: {
      background: 'rgba(0, 0, 0, 0.3)',
    },
  };`

That's the reason I'm stuck.